gatsby/plugins/gatsby-source-bnf/src/node-creation/drugs.ts

```diff
@@ -35,6 +62,22 @@ export const createDrugNodes = (
 				message: constituentDrugs.message,
 				constituents: constituentDrugs.constituents.map((d) => d.sid),
 			},
+			medicinalForms: {
+				initialStatement,
+				specialOrderManufacturersStatement,
+				medicinalForms: forms?.map((medicinalForm) => {
```

Comment:
as `forms` is possibly undefined, I _think_ we should do `forms?.map(...) || []` to make sure we always have at least an empty array (rather null/undefined).